python/nav/web/macwatch/forms.py

```diff
@@ -37,11 +35,14 @@ class MacWatchForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super(MacWatchForm, self).__init__(*args, **kwargs)
-        self.helper = FormHelper()
-        self.helper.form_action = '.'
-        self.helper.layout = Layout(
-            Fieldset('Add mac to watch list', 'macaddress', 'description'),
-            ButtonHolder(NavSubmit('submit', 'Add')),
```

Comment:
Since the `NavSubmit` here was wrapped in a `ButtonHolder` class, which basically just wraps the button in a `<div class="button-holder">`, this is lacking in the new uncrispified version. But it does look the same, so I don't think it matters.

What do you think @podliashanyk?

Reply:
yeah this is a weird one, it seems to look and act exactly the same :shrug: